pallets/rmrk-core/src/functions.rs

```diff
@@ -693,13 +699,19 @@ impl<T: Config>
 		)?;
 
 		Self::deposit_event(Event::NFTSent {
-			sender,
+			sender: sender.clone(),
 			recipient: new_owner,
 			collection_id,
 			nft_id,
 			approval_required,
 		});
 
+		// Defaults to true, but can be implemented downstream for custom logic
+		ensure!(
+			T::CheckAllowTransfer::post_check(&sender, &new_owner_account, &collection_id, &nft_id),
```

Comment:
In fact, the post check function is not use to check if the transfer is allowed, we may call CheckAllowTransfer as TransferHooks or something else maybe?